openshift/db/files/emergency-restore.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+#
+# Emergency Database Restore Script
+# Use this for restoring to a completely fresh PostgreSQL instance
+# that doesn't have any existing roles.
+#
+# Usage: ./emergency-restore.sh <backup-file.sql.gz> <database-name>
+#
+
+set -e
+
+if [[ $# -ne 2 ]]; then
+    echo "Usage: $0 <backup-file.sql.gz> <database-name>"
+    echo "Example: $0 /backups/daily/2025-11-25/met-patroni-app_2025-11-25_10-34-33.sql.gz app"
+    exit 1
+fi
+
+BACKUP_FILE="$1"
+DATABASE="$2"
+
+source /vault/secrets/met-patroni
+export PGPASSWORD="$MET_PATRONI_PASSWORD"
+
+echo "========================================"
+echo "Emergency Database Restore"
+echo "========================================"
+echo "Backup file: $BACKUP_FILE"
+echo "Target database: $DATABASE"
+echo ""
+
+# Check if target database already exists and has data
+DB_EXISTS=$(psql -h met-patroni -U "$MET_PATRONI_USER" -d postgres -tc "SELECT 1 FROM pg_database WHERE datname = '$DATABASE'" | grep -q 1 && echo "yes" || echo "no")
```

Comment:
Should we consider making the host dynamic via env vars? For example, met is already outdated.